packages/grid/scss/_mixins.scss

```diff
@@ -44,16 +44,23 @@
 }
 
 /// Define the width of the column for a given span and column count.
+/// A width of 0 will hide the column entirely.
```

Comment:
Could we move the 0 span enhancements under a separate PR? 🙏 Definitely a great feature but might help to pull it out 👍

Reply:
reverted in 9afd776, d6e6bf9, 54d56d5.

PR here #5052

(there's probably a more eloquent way to transfer 2 commits to a new PR that I'm unfamiliar with, but just creating new commits and referencing this is wayyyy quicker 😄)